server/webhook.go

```diff
@@ -19,13 +19,20 @@ type Webhook interface {
 	PostNotifications(p *Plugin) ([]*model.Post, int, error)
 }
 
+type webhookEvent struct {
+	field string
+	from  string
+	to    string
+}
+
 type webhook struct {
 	*JiraWebhook
 	eventMask     uint64
 	headline      string
 	text          string
 	fields        []*model.SlackAttachmentField
```

Comment:
are these even used now?

Reply:
fields? Yes, for the create_issue event

Reply:
We should probably consolidate... why have 2 collections of fields, and there are more fields to display in Create. Just no past values. But not in this PR.